### PR TITLE
feat(ingest): add new Amplitude ingestion source

### PR DESCRIPTION
**Pull request by gmcgoldrick-r7**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/tests/integration/amplitude/setup/event_type.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "data": [
+        {
+            "event_type": "test_event",
+            "category": {
+                "name": "test_category"
+            },
+            "description": "This is a test event and category"
+        }
+    ]
+}
```
Comment (asikowitz) [inline metadata-ingestion/tests/integration/amplitude/setup/event_type.json:12]: Can you add multiple events to the test data?

File: metadata-ingestion/docs/sources/amplitude/amplitude.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Metadata Ingestion Quickstart
```
[the remark this answers is not in the document]
Comment (gmcgoldrick-r7) [reply metadata-ingestion/docs/sources/amplitude/amplitude.md:1]: Done

File: metadata-ingestion/docs/sources/amplitude/amplitude.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+* an account in Amplitude
+* at least one project within that account that contains data
+* api keys and secret keys for each project to ingest
+
+## Install the Plugin(s)
+
+Run the following commands to install the relevant plugin(s):
+
+`pip install 'acryl-datahub[amplitude]'`
```
[the remark this answers is not in the document]
Comment (gmcgoldrick-r7) [reply metadata-ingestion/docs/sources/amplitude/amplitude.md:15]: Done

File: metadata-ingestion/docs/sources/amplitude/amplitude.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  type: "amplitude"
+  config:
+    connect_uri: "https://amplitude.com/api/2"
+    projects:
+      - name: project name
+        description: project description
+        api_key: project api key
+        secret_key: project secret key
+sink:
+  # sink configs
```
[the remark this answers is not in the document]
Comment (asikowitz) [reply metadata-ingestion/docs/sources/amplitude/amplitude.md:32]: I don't think you've pushed this change or the others you responded to below

**Comment by asikowitz** [comment]
Hey @gmcgoldrick-r7, are you still working on this? Do you need any help in responding to my comments? Thanks!

**Comment by gmcgoldrick-r7** [comment]
> Hey @gmcgoldrick-r7, are you still working on this? Do you need any help in responding to my comments? Thanks!

Hi @asikowitz, apologies - I had to suddenly pivot to something else and work on that. I'll be back to this in the next week or so. Thanks for checking in!

**Comment by gmcgoldrick-r7** [comment]
No longer required.

**Pull request closed without merging**